docs/apache-airflow/core-concepts/dags.rst

```diff
@@ -190,18 +190,20 @@ DAGs do not *require* a schedule, but it's very common to define one. You define
     with DAG("my_daily_dag", schedule="@daily"):
         ...
 
-The ``schedule`` argument takes any value that is a valid `Crontab <https://en.wikipedia.org/wiki/Cron>`_ schedule value, so you could also do::
+There are various valid values for the ``schedule`` argument::
 
     with DAG("my_daily_dag", schedule="0 0 * * *"):
         ...
 
-.. tip::
+    with DAG("my_one_time_dag", schedule="@once"):
+        ...
 
-    For more information on ``schedule`` values, see :doc:`DAG Run <dag-run>`.
+    with DAG("my_continuous_dag", schedule="@continuous"):
+        ...
 
-    If ``schedule`` is not enough to express the DAG's schedule, see :doc:`Timetables </howto/timetable>`.
-    For more information on ``logical date``, see :ref:`data-interval` and
-    :ref:`faq:what-does-execution-date-mean`.
+.. tip::
+
+    For more information different types of scheduling, see :doc:`authoring-and-scheduling/index`.
```

Comment:
I need some help here. I'm not sure why sphinx isn't recognizing this reference.

Reply:
You miss forward slash at the beginning. If in doubt, look at other examples where `:doc:` is used.